react/tic-tac-toe/src/constants/routes.js

```diff
@@ -6,5 +6,9 @@ export const ROUTES = {
   game: {
     path: '/game',
     page: 'Game'
+  },
+  settings: {
```

Comment:
You should use this paths in the Home Routes where you render the private routes